### PR TITLE
Add Mountfile handling function

### PR DESCRIPTION
**Pull request by macropin**
Add Mountfile support

### CODE REVIEW

File: functions/run_mountfile.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# run_mountfile [FILENAME] [DATADIR]
+# run_mountfile Mountfile /srv/remote
+run_mountfile() {
+  # Mount all dirs specified in Mountfile
+  local mountfile="${1:-'Mountfile'}"
+  local data="${2:-'/srv/remote'}"
```
Comment (trnubo) [inline functions/run_mountfile.sh]: These defaults don't seem to work
```
root@b1278dd46903:/var/www# run_mountfile 
Mountfile not found
root@b1278dd46903:/var/www# run_mountfile Mountfile 
Data dir not found
```

**Comment by trnubo** [comment]
Should there be protection against the target_dir traversing into the data dir and causing the data/remote directory being deleted?

```
root@b1278dd46903:/var/www# mkdir /srv/remote/mycontent
root@b1278dd46903:/var/www# touch /srv/remote/mycontent/file1
root@b1278dd46903:/var/www# run_mountfile Mountfile /srv/remote
Mounting remote path mycontent => ../../srv/remote/mycontent
root@b1278dd46903:/var/www# ls -l /srv/remote/mycontent/
ls: cannot access /srv/remote/mycontent/: Too many levels of symbolic links
root@b1278dd46903:/var/www# ls -l /srv/remote/mycontent 
lrwxrwxrwx. 1 root root 21 Jul  2 04:22 /srv/remote/mycontent -> /srv/remote/mycontent
root@b1278dd46903:/var/www# cat Mountfile 
mycontent : ../../srv/remote/mycontent
```

File: functions/run_mountfile.sh (hunk 1 above)
Comment (macropin) [reply functions/run_mountfile.sh]: Resolved with latest commit

**Comment by macropin** [comment]
Please try now... Additional checks have been added
```
root@672f13737895:/var/www# echo "mycontent : ../../srv/remote/mycontent" > Mountfile
root@672f13737895:/var/www# run_mountfile Mountfile /srv/remote
Error: Target outside working directory!
```

**Comment by trnubo** [comment]
LGTM

**Pull request merged**